spock-core/src/main/java/org/spockframework/mock/runtime/MockSettings.java

```diff
@@ -0,0 +1,72 @@
+package org.spockframework.mock.runtime;
```

Comment:
This class does not adhere the formatting rule, code positions e.g. fields before constructor, qualified names  instead of imports etc.
Nor is it Immutable, nor has a toString (for debugging purpose) + equal+ hashcode for POJO, etc.

So this new class does not make the original code more readable nor is this class maintainable.